org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "command": "qelectrotech",
+  "rename-desktop-file": "qelectrotech.desktop",
+  "rename-appdata-file": "qelectrotech.appdata.xml",
+  "rename-icon": "qelectrotech",
+  "copy-icon": true,
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
+    "--device=dri",
+    "--share=ipc",
+    "--filesystem=host"
```

Comment:
The `--filesystem=host` is a little too permissive for this kind of app.  
A User who blocks this permission will hit an issue with the app home folder not being kept so maybe add a comment somewhere about `~/.qet`?  or maybe just add `- --persist=.qet`?

Reply:
Need input from @scorpio810 on this.

Reply:
Yes, what do you need full host permissions for? As @tinywrkb says we should allow the absolute minimum surface that lets the app access its configuration (ideally that uses xdg-config dirs so it doesn't need a permission at all, but a --persist=.qet could work too - and then as a modern Qt app I'd hope that it should be able to use the file portal to open and save files anywhere on the system as required without extra permissions.

You can test this yourself locally with --nofilesystem=host and the persist flag mentioned above on the flatpak CLI - or use an app like flatseal https://flathub.org/apps/details/com.github.tchx84.Flatseal to experiment with the permissions and minimise them.

Certainly host permissions require explicit justification that is core to the purpose of the app.

Reply:
I think qelectrotech doesn't use portals. It's not possible to save a project if you take away the host permission.